hack/verify-govet.sh

```diff
@@ -23,7 +23,55 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false $test_dir
+  go tool vet -all $test_dir
+  if [ "$?" -ne 0 ]
+  then
+    FAILURE=true
+  fi
+done
+
+# For the sake of slowly white-listing `shadow` checks, we need to keep track of which
+# directories we're searching through. The following are all of the directories we care about:
+ALL_DIRS='./cmd
+./examples
+./images
+./pkg/api
+./pkg/assets
+./pkg/auth
+./pkg/authorization
+./pkg/build
+./pkg/client
+./pkg/cmd
+./pkg/config
+./pkg/controller
+./pkg/deploy
+./pkg/diagnostics
+./pkg/dns
+./pkg/dockerregistry
+./pkg/generate
+./pkg/gitserver
+./pkg/image
+./pkg/ipfailover
+./pkg/oauth
+./pkg/project
+./pkg/route
+./pkg/router
+./pkg/sdn
+./pkg/security
+./pkg/service
+./pkg/serviceaccounts
+./pkg/template
+./pkg/user
+./pkg/util
+./pkg/version
+./plugins
```

Comment:
I'd rather blacklist so we can delete exceptions as we clean them up and new packages will get enforced.